01_science_technology_and_epistemology/01_science_technology_and_epistemology.jl

```diff
@@ -8,72 +8,175 @@ using InteractiveUtils
+Technology is the thing that allows us as humans to transform reality. And here it is important to clarify that technology is not something "new". 
+It has been with us since the very beginning of the human era, and it is really what makes us different from other animals. 
 
-How? Well, when we say that we use technology to transform the reality, we are referring to the fact that we do so through the *expansion of our capabilities*. From taking a long stick with a point to reach a fruit high up in a tree, passing through transforming an entire landscape by building a huge electric dam, to the creation of artificial intelligence that helps us solve our deepest puzzles. With technology we transform our *experience* in the reality in which we live.
+How? Well, when we say that we use technology to transform the reality, we are referring to the fact that we do so through the *expansion of our capabilities*.
+From taking a long stick with a point to reach a fruit high up in a tree, passing through transforming an entire landscape by building a huge electric dam, to the creation of artificial intelligence that helps us solve our deepest puzzles. 
+With technology we transform our *experience* in the reality in which we live.
 
-The current understanding of how knowledge is produced is very strange and even counterproductive for the development of humanity. Nowadays, there is a kind of *linear thinking* model in which we take for granted that the process of knowledge production is very definite: We first make academic and theoretical discoveries and only then, with all of that formalization made, we can start thinking of putting it to practical use in the real world. This is a completely wrong way of thinking and is holding us back enormously, causing important funds to be misused.
+The current understanding of how knowledge is produced is very strange and even counterproductive for the development of humanity. 
+Nowadays, there is a kind of *linear thinking* model in which we take for granted that the process of knowledge production is very definite: We first make academic and theoretical discoveries and only then, with all of that formalization made, we can start thinking of putting it to practical use in the real world. 
+This is a completely wrong way of thinking and is holding us back enormously, causing important funds to be misused.
 
+If you start looking at the technology/innovation history you may gradually begin to notice a repeating pattern: There is a constant differentiation between the people who *do* and the people who *watch* and then narrate. 
+"The History Written by the Losers" would [Nassim Nicholas Taleb](https://en.wikipedia.org/wiki/Nassim_Nicholas_Taleb) said, the people that are doing stuff don't have time for beauty writing. But the thing does not stop there. 
```

Comment:
> "The History Written by the Losers" would [Nassim Nicholas Taleb](https://en.wikipedia.org/wiki/Nassim_Nicholas_Taleb) said,

fix sentence

Reply:
>"The History Written by the Losers" would Nassim Nicholas Taleb said,

this reference may be unclear for someone who hadn't read Taleb's book. Also, a suggestion of a better syntax is

>"The History Written by the Losers" as Nassim Nicholas Taleb says,

Reply:
Yup, I was referring to the syntax, not the link :)

Reply:
As Nassim Nicholas Taleb wrote in "The History Written by the Losers"...